client/src/templates/Challenges/redux/index.js

```diff
@@ -239,6 +242,10 @@ export const reducer = handleActions(
       ...state,
       challengeFiles: payload
     }),
+    [types.saveCode]: (state, { payload }) => ({
```

Comment:
Why update the store? Does the file change?

Reply:
Yeah this was not necessary.